packages/openactive-integration-tests/test/types/chakram/index.d.ts

```diff
@@ -22,9 +22,13 @@ declare module 'chakram' {
     responseTime: number;
   }
 
-  type RequestOptions = Pick<Options, 'headers' | 'body'>;
+  // `url` is required in request's options, but is not needed in chakram,
+  // because `url` is provided as a separate argument to params
+  export type RequestOptions = Omit<Options, 'url'>;
```

Comment:
Yes, chakram's "params" arg has the same type as the `request` library. They state so here: http://dareid.github.io/chakram/jsdoc/module-chakram.html#.request. Though with the apparent exception that `url` is missing. I expect there are more exceptions tbh but nothing's broken so far